admin_app/src/app/playground/components/PlaygroundComponents.tsx

```diff
@@ -310,16 +361,51 @@ const MessageBox = (message: Message) => {
             ? message.content
             : renderResults(message.content)}
         </Typography>
-        {message.hasOwnProperty("json") && (
-          <Link
-            onClick={toggleJsonModal}
-            variant="caption"
-            align="right"
-            underline="hover"
-            sx={{ cursor: "pointer" }}
+        {message.type == "response" && (
+          <Box
+            style={{
+              marginTop: "5px",
+              display: "flex",
+              justifyContent: "flex-end",
+              alignItems: "center",
+            }}
           >
-            {"<json>"}
-          </Link>
+            {message.json.hasOwnProperty("feedback_secret_key") && (
```

Comment:
Only display if the json contains `feedback_secret_key`

Reply:
Suggest using ternary operator

https://alexkondov.com/tao-of-react/#conditional-rendering